Services/PrivacySecurity/classes/Setup/class.ilPrivacySecuritySetupAgent.php

```diff
@@ -41,7 +41,7 @@ public function getArrayToConfigTransformation() : Refinery\Transformation
     {
         return $this->refinery->custom()->transformation(function ($data) {
             return new \ilPrivacySecuritySetupConfig(
-                (bool) $data["https_enabled"]
+                (bool) $data["https_enabled"] ?? false
```

Comment:
Please can you change this to (bool) ($data["https_enabled"] ?? false) for PHP 8 compatability.

Reply:
Indeed I can =)